### ISSUE TITLE
Sonoff SNZB-01, SNZB-02 - Pairing works, but devices are not functioning

### ISSUE THREAD

**Issue by bekulini** [ioBroker/ioBroker.zigbee#863]
### Hardware in use:
Raspberry pi 4, with 4Gb, IObroker and CC2531 as ZigBee coordinator (firmware 1.2 with revision date: 20190608).  

### Issue description:
I purchased several SNZB-01 and SNZB-02 devices and started pairing with very first SNZB-01. Pairing worked and device was functioning as designed and currently in use. After I couldn't get any further SNZB-01 or any SNZB-02 working, all with the same issue:
(1) Pairing of devices works without any errors (see screenshots)
(2) After pairing is complete I can device in ZigBee Instance and also under objects (see screenshot)
(3) however, under objects I can only see that the device is available, but the rest is empty. So no further information on temperature, humidity, battery status or in case of SNZB-01 (one, double, long) button click available. 

Am I doing something wrong? Can someone please help with found the root of the issue? Interesting that the very first SNZB-01 worked and is still functioning. 

I tried already several things, but nothing helped :-(:
- tried 5 different SNZB-01 devices, so I assume the devices are not broken;
- tried different CC2531 USB stick with different versions of firmware (ZigBee 3.0, source router firmware)
- tried completely new IOBroker setup on different Raspberry Pi
..currently trying to setup Home Assistant in order to check if the devices would work under Home Assistant OS.

Any help, hint is very much appreciated. 

![Instanz_Screenshot](https://user-images.githubusercontent.com/74904830/100074443-0a90a100-2e3f-11eb-8ec1-f3f11a9144d3.PNG)
![Log_Screenshot](https://user-images.githubusercontent.com/74904830/100074446-0bc1ce00-2e3f-11eb-9280-b5eef23023fb.PNG)
![ZigBeeObject](https://user-images.githubusercontent.com/74904830/100074448-0bc1ce00-2e3f-11eb-90c5-8e0e25eb9492.PNG)

**Comment by kirovilya** [comment]
try to update adapter from github
switch log to debug and show what's in log when devices in action

**Comment by bekulini** [comment]
Good afternoon Ilya, thank you for your prompt reply! I have now installed ZigBee 1.3.9dev version from github, enabled logging level "debug" and paired SNZB-01 with ID "00124b001eab1e04" (after force removing before).

I am attaching log file, which probably contains too much information :-( (sorry, I am new in area) but you can find this pairing procedure starting at 2020-11-24 12:23:49.199 and finishing at 2020-11-24 12:24:50.461. 

[iobroker.2020-11-24.log.txt](https://github.com/ioBroker/ioBroker.zigbee/files/5589756/iobroker.2020-11-24.log.txt)

**Comment by kirovilya** [comment]
I look at the logs and really don't see any messages from the button. The temperature sensor must also be configured.
Bad batteries? or far from the coordinator?
Delete their networks - long press the pairing button on the device and then re-connect them to the network.
do not use force deletion if devices are available.

**Comment by bekulini** [comment]
Privet Ilya! Even if it shows that devices are available, I can delete devices without "force deletion". This is error, which I get if I try to delete without force deletion: 
![image](https://user-images.githubusercontent.com/74904830/100147966-5faed000-2e9c-11eb-91a4-b8ce0bd15e10.png)

Batteries are fine, I have already used the battery of only one (and very first) SNZB-01, which is connected and working properly. devices were not even 1 meter away from coordinator... :-( 

Do you have any further ideas what could be causing my problem? is it possible to "copy" configuration of one working device? Could this be caused by fact that coordinator is close to TV and wifi router? :-( I am running out of options.

Spasibo bolshoe!

**Comment by kirovilya** [comment]
For delete - "long press the pairing button on the device", not in adapter admin.
Other devices work fine?

**Comment by bekulini** [comment]
Hello Ilya, today I have relocated my raspberri pi, installed second zigbee adapter with different CC2531 and paired again several SNZB-01 and 02, and finally I could identify the root of my silly mistake :-(  So that you understand, I will short background how raspberry pi is located in my case:
it is hanging behind TV, however USB CC2531 was set with longer USB cable slightly above TV, but still a bit behind it. while I was trying to pair devices, I was standing close to CC2531, however holding SNZBs in my hand at lower level so that signal had to go through TV and this was apparently an issue... Now I am using CC2531 with Antenna and I have ensured that Antenna is definitely above TV, so that there is "good link" between coordinator and first devices. So, it was purely my stupid mistake :-(

Thank you for your time and investigation :-) happy to be part of this forum here now.
